ion-schema/src/isl/isl_type_reference.rs

```diff
@@ -126,6 +131,7 @@ impl IslTypeRefImpl {
         value: &Element,
         inline_imported_types: &mut Vec<IslImportType>,
     ) -> IonSchemaResult<Self> {
+        use crate::isl::isl_type_reference::NullabilityModifier::*;
```

Comment:
minor suggestion—for readability, it might be helpful to refactor something like this:
```suggestion
        use crate::isl::isl_type_reference::NullabilityModifier::*;
        let nullability = if value.has_annotation("nullable") {
            if isl_version == IslVersion::V1_0 { Nullable } else {
                return invalid_schema_error("`nullable::` modifier is not supported since ISL 2.0")
            }
        } else if value.has_annotation("$null_or") {
            if isl_version != IslVersion::V1_0 { NullOr } else {
                return invalid_schema_error("`$null_or::` modifier is not supported before ISL 2.0")
            }
        } else {
            Nothing
        }
```

This way, we can eliminate some of the duplicate code in the match arms. See comment below, for example.